components/tips/view.coffee

```diff
@@ -8,7 +8,10 @@ module.exports = class TipView extends Backbone.View
     'click': 'onClick'
     'click .js-close' : 'close'
 
-  initialize: ({ @user }) -> #
+  initialize: ({ @user }) ->
+    @dismisser = new Dismisser
+      key: @model.id
```

Comment:
What's model id evaluate to?

Reply:
Something like so: https://github.com/aredotna/ervell/blob/master/apps/user/tips.coffee#L3

Reply:
ah cool. got it